benchmarks/protobuf-java-benchmarks/build.gradle.kts

```diff
@@ -33,7 +33,7 @@ protobuf {
 }
 
 dependencies {
-    implementation(project(":benchmarks:util"))
+    implementation(project(":benchmarks:benchmarks-util"))
```

Comment:
Having an Android plugin applied anywhere in the project forces unique names for every subproject.